packages/@ember/service/tests/service_test.js

```diff
@@ -1,47 +1,51 @@
 import Service, { inject, service } from '@ember/service';
 import EmberObject from '@ember/object';
 import { buildOwner, runDestroy } from 'internal-test-helpers';
-import { moduleFor, AbstractTestCase } from 'internal-test-helpers';
+import { moduleFor, AbstractTestCase, expectDeprecation } from 'internal-test-helpers';
 
 moduleFor(
   'inject - decorator',
   class extends AbstractTestCase {
     ['@test works with native decorators'](assert) {
```

Comment:
I think you need `testUnless` for these tests, ala https://github.com/emberjs/ember.js/blob/3a1806a1a7d1db007f72e452175294479a6e8113/packages/@ember/-internals/glimmer/tests/integration/application/actions-test.js#L61

Reply:
Thanks!